shiny/express/_output.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import contextlib
+import sys
+from contextlib import AbstractContextManager
+from typing import Callable, ParamSpec, TypeVar, cast, overload
+
+from .. import ui
+from ..render.transformer import OutputRenderer
+
+__all__ = (
+    "output_args",
+    "suspend_display",
+)
+
+OT = TypeVar("OT")
+P = ParamSpec("P")
+R = TypeVar("R")
+CallableT = TypeVar("CallableT", bound=Callable[..., object])
+
+
+def output_args(
```

Comment:
No need to do this immediately since this PR is time-sensitive, but it will be helpful to have example apps.

Reply:
Let's do that in the flat-mode branch, cause you can't really do anything interesting with them until then, unless you have quarto